src/tools/dart2native.md

```diff
@@ -24,6 +24,13 @@ An **AOT snapshot** doesn't include the Dart runtime.
 Consider using snapshots if you're distributing multiple programs
 and disk space is limited.
 
+{{site.alert.note}}
```

Comment:
I think I'd prefer this at the bottom of the page. It's really only applicable to a very, very small user base, so should not be placed so prominently

Reply:
I'll put that in a new PR.